### ISSUE TITLE
Textures inconsistent

### ISSUE THREAD

**Issue by iam-cult** [CyanideX/Unity#213]
Textures of things like stone and ores are only sometimes changed, and torches placed on walls use the vanilla texture

**Comment by CyanideX** [comment]
This issue does not follow the provided template. These templates exist for a reason: it avoids having me ask the same questions over and over again. First and foremost, what is the version of Minecraft and Unity that you are experiencing issues with? This goes in the title of your issue.

**Describe the Issue**
A clear and concise description of what the issue is. So far, I see that stone and ores are sometimes changed. Does that mean they are sometimes vanilla? Are you referring to block variants which are subtly different textures, designed to break up the obvious tiling of blocks?

**Screenshots**
If applicable, add screenshots to help explain your problem. THIS IS IMPORTANT! Especially in situations like this... show me what is going on. Attach a screenshot.

**Additional Context**
Add any other context about the problem here. Do you happen to have any mods installed? Do you have any other resource packs enabled? Are you using OptiFine?

To sum up: use issue templates, provide as much info as possible (specifically version info), and attach sample screenshots. Once you get me this info, I will happily look into the issue for you. Thank you. 🙂

I will also look into the torch issue. Seems simple enough.

**Comment by CyanideX** [comment]
Fixed torches in fee099b!

![Torches](https://user-images.githubusercontent.com/3106700/88009677-ee16dd80-cac7-11ea-9179-06505c784916.png)

**Comment by CyanideX** [comment]
Closing this for now since I'm assuming your issue with textures is that they're using variants. Please feel free to reopen if you can share a screenshot and provide further details.

**Comment by iam-cult** [comment]
I didn't have time to take screenshots, I'll try to get some today :)

**Comment by iam-cult** [comment]
If the issue is still present I will open a new issue
